ndpyramid/testing.py

```diff
@@ -0,0 +1,34 @@
+import mercantile
+import numpy as np
+import xarray as xr
+
+
+def _bounds(ds):
+    left = ds.x[0] - (ds.x[1] - ds.x[0]) / 2
+    right = ds.x[-1] + (ds.x[-1] - ds.x[-2]) / 2
+    top = ds.y[0] + (ds.y[1] - ds.y[0]) / 2
+    bottom = ds.y[-1] - (ds.y[-1] - ds.y[-2]) / 2
+    return (left.data, bottom.data, right.data, top.data)
```

Comment:
@andersy005 @norlandrhagen do either of you know a better way to get the bounding box from a regularly spaced dataset with coordinates known to be 'x' and 'y'?